### PR TITLE
periodic-branch-protector: 6h timeout

### PR DESCRIPTION
**Pull request by hongkailiu**
Follow up https://coreos.slack.com/archives/CHY2E1BL4/p1613401229199600

Before we figure out why it takes so long, try with a bigger value of timeout.

/cc @petr-muller

### CODE REVIEW

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -660,12 +660,14 @@ periodics:
 - agent: kubernetes
   cluster: app.ci
   decorate: true
+  decoration_configs:
+    timeout: 6h0m0s
```
Comment (petr-muller) [inline ci-operator/jobs/infra-periodics.yaml:664]: This is `decoration_config`

**Review by petr-muller — APPROVED** [review]
It's a bandaid, but we should really look into why this takes this long :/

File: ci-operator/jobs/infra-periodics.yaml (hunk 1 above)
Comment (petr-muller) [reply ci-operator/jobs/infra-periodics.yaml:664]: Opps this is stale

Comment (hongkailiu) [reply ci-operator/jobs/infra-periodics.yaml:664]: Updated.

**Pull request merged**